frontend/src/components/Query/Column/ColumnExpressionPicker.vue

```diff
@@ -72,8 +72,21 @@
 				:options="columnTypes"
 			/>
 		</div>
+		<div v-if="showDateFormatOptions" class="mt-2 text-sm text-gray-600">
+			<div class="font-light">Date Format</div>
+			<Autocomplete
+				v-model="expression.dateFormat"
+				:options="dateFormats"
+				placeholder="Select a date format..."
+			/>
+		</div>
 		<div class="mt-4 text-sm text-gray-600">
-			<Input type="checkbox" label="Group By" v-model="expression.groupBy" />
+			<Input
+				type="checkbox"
+				:label="!showDateFormatOptions ? 'Group By' : 'Group By (Unable to group by Date/Datetime expressions at this time).'"
+				v-model="expression.groupBy"
+				:disabled="showDateFormatOptions"
+			/>
```

Comment:
```suggestion
			<Input
				v-if="!showDateFormatOptions"
				type="checkbox"
				label="Group By"
				v-model="expression.groupBy"
			/>
```

Reply:
So, `group by` doesn't work with `aggregated` fields
Until I figure out, let's disable group by for date expressions